lsstseries/analysis/structurefunction2.py

```diff
@@ -1,38 +1,52 @@
 from scipy.stats import binned_statistic
 import numpy as np
+import pandas as pd
 
 
-def calc_sf2(time, flux, err, bins, band, band_to_calc=None):
+def calc_sf2(lc_id, time, flux, err, band,
+             band_to_calc=None, combine=False, method='size', sthresh=100):
```

Comment:
It should still be possible to manually specify bins. I like the bins methods that you implemented, but they should not remove the possibility to manually add bins.

Reply:
Seems reasonable, will do